server/src/controllers/user.js

```diff
@@ -317,16 +321,23 @@ export const getProfile = async (request, response) => {
     const { token } = request.cookies;
 
     if (token) {
-      jwt.verify(token, secret, {}, (error, info) => {
+      jwt.verify(token, secret, {}, async (error, info) => {
         if (error) {
           response
             .status(498)
             .json({ success: false, message: 'Invalid token' });
         } else {
-          response.status(200).json({
-            success: true,
-            user: info,
-          });
+          const user = await User.findById(info.id);
+          if (!user) {
+            response
+              .status(200)
+              .json({ success: true, message: 'User not found' });
```

Comment:
Some clarification is required here. I don't quite understand why if a user is not found, the server responds with status 200 and success: true